ansible/roles/tower-user-create/tasks/main.yml

```diff
@@ -8,18 +8,20 @@
 
 - name: Add tower user
   tower_user:
-     username: "{{ item.user }}"
-     password: "{{ item.password | default('change_me') }}"
-     email: "{{ item.email | default('rhpds-admins@redhat.com') }}"
-     first_name: "{{ item.firstname | default(item.user) }}"
-     last_name: "{{ item.lastname | default(item.user) }}"
-     superuser: "{{ item.superuser | default('no')}}"
+     username: "{{ user.user }}"
+     password: "{{ user.password | default('change_me') }}"
+     email: "{{ user.email | default('rhpds-admins@redhat.com') }}"
+     first_name: "{{ user.firstname | default(user.user) }}"
+     last_name: "{{ user.lastname | default(user.user) }}"
+     superuser: "{{ user.superuser | default('no')}}"
```

Comment:
isn't `user` even more generic than `item` ? what happens if `user` is passed as extra-var ?